iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -100,6 +100,13 @@ Fact.prototype.readableValue = function() {
         /* Replace runs of whitespace (including nbsp) with a single space */
         v = html.text().replace(/[\u00a0\s]+/g, " ").trim();
     }
+    else if (this.isEnumeration()) {
+        var labels = [];
+        for (const qn of v.split(' ')) {
+            labels.push(this._report.getLabel(qn, 'std') || qn);
+        }
+        v = labels.join(', ');
```

Comment:
Do you think people would prefer to see this as a bulleted list rather than a comma-separated list?  If we don't yet have enough experience with enum set, I'm fine with this for now.

Reply:
Just had a quick chat with Dave Shaw on use in the US-GAAP taxonomy, and it sounds like most cases will actually be a single item, but where is more than one, a bulleted list would be clearer, so I'll make that change.